### ISSUE TITLE
bug: VGG16 without pretrained=True option?

### ISSUE THREAD

**Issue by CM-BF** [avinashpaliwal/Super-SloMo#54]
In train.py line 111 `vgg16 = torchvision.models.vgg16()`, there may be a missing option: `pretrained=True`.
By default, `pretrained=False`.
Thus, perhaps perceptual loss does not work.

**Comment by QianyeYang** [comment]
just found the same problem

**Comment by BostonLobster** [comment]
Also found it just now, could you please fix it?@avinashpaliwal

**Comment by avinashpaliwal** [comment]
Thanks for pointing out the issue.
